sdk/ai/Azure.AI.Projects/tests/Azure.AI.Projects.Tests.csproj

```diff
@@ -1,35 +1,37 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-
     <!-- We don't care about XML doc comments on test types and members -->
     <NoWarn>$(NoWarn);CS1591</NoWarn>
   </PropertyGroup>
 
-  <!-- Reference the Client Library -->
+  <!-- Reference Foundry libraries via project -->
   <ItemGroup>
     <ProjectReference Include="$(AzureCoreTestFramework)" />
     <ProjectReference Include="..\src\Azure.AI.Projects.csproj" />
+    <ProjectReference Include="..\..\Azure.AI.Projects.OpenAI\src\Azure.AI.Projects.OpenAI.csproj" />
+    <ProjectReference Include="..\..\Azure.AI.Agents.Persistent\src\Azure.AI.Agents.Persistent.csproj" />
+    <ProjectReference Include="..\..\Azure.AI.Inference\src\Azure.AI.Inference.csproj" />
   </ItemGroup>
-  
+
   <ItemGroup>
-    <PackageReference Include="Azure.AI.Agents.Persistent" VersionOverride="1.0.0" />
-    <PackageReference Include="Azure.AI.OpenAI" VersionOverride="2.2.0-beta.4" />
-    <PackageReference Include="Azure.AI.Inference" />
-    <PackageReference Include="NUnit" />
-    <PackageReference Include="NUnit3TestAdapter" />
+    <PackageReference Include="Azure.AI.OpenAI" />
+    <PackageReference Include="NUnit" VersionOverride="4.4.0" />
+    <PackageReference Include="NUnit3TestAdapter" VersionOverride="5.1.0"/>
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="Moq" />
     <PackageReference Include="Azure.ResourceManager" />
     <PackageReference Include="Azure.ResourceManager.CognitiveServices" />
     <PackageReference Include="OpenTelemetry" />
     <PackageReference Include="OpenTelemetry.Exporter.Console" />
     <PackageReference Include="Azure.Monitor.OpenTelemetry.Exporter" />
+    <PackageReference Include="Microsoft.ClientModel.TestFramework" />
     <PackageReference Include="System.Linq.Async" />
```

Comment:
This needs to change to `System.Linq.AsyncEnumerable`.   The former package is not supported in `net10`